containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
+    && apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+
+RUN mkdir -p ${APP_ROOT}
+COPY ./ ${APP_ROOT}/
+
+WORKDIR ${APP_ROOT}
+
+RUN gem install bundler
+
+ENV RACK_ENV=production RAILS_ENV=production
+
+RUN bundle config set without 'development private test' \
+    && bundle config set deployment 'true' \
+    && bundle config set with 'sqlite mysql postgres' \
+    && bundle install \
+    && yarn install
```

Comment:
`yarn install` should be in it's own step/layer cause of caching.

Reply:
fine